### ISSUE TITLE
Fix minimum version of Hoa/ruler

### ISSUE THREAD

**Issue by agallou** [atoum/ruler-extension#16]
In the composer.json hoa/ruler version is `~1.0`

But the version is not "1.14.12.10", 

With this one we have this error because the interpret version does not exists :

```
Fatal error: Call to undefined method Hoa\Ruler\Ruler::interpret() in /home/agallou/Projets/b2c/magento/vendor/atoum/ruler-extension/classes/ruler.php on line 26
```

If we update hoa/ruler

```
  - Removing hoa/ruler (1.14.12.10)
  - Installing hoa/ruler (1.15.02.05)
```

The filter option works.

The mimum version of hoa/ruler should be in the composer.json file.

**Comment by agallou** [comment]
this issue is now fixed by #22

**Comment by Metalaka** [comment]
@Hywan Fixed by #22

**Comment by Hywan** [comment]
@agallou Can you close this issue please?
